commit fd7efa18bdfdb576c36fc8e09a0673a68fd86f71
Author: Dimitry Andric <dim@FreeBSD.org>
Date:   Tue Oct 19 23:54:48 2021 +0200

    graphics/qt5-wayland: fix build with clang 13 by adding <array>

diff --git a/graphics/qt5-wayland/files/patch-src_hardwareintegration_compositor_linux-dmabuf-unstable-v1_linuxdmabuf.h b/graphics/qt5-wayland/files/patch-src_hardwareintegration_compositor_linux-dmabuf-unstable-v1_linuxdmabuf.h
new file mode 100644
index 000000000000..eae8123a0da4
--- /dev/null
+++ b/graphics/qt5-wayland/files/patch-src_hardwareintegration_compositor_linux-dmabuf-unstable-v1_linuxdmabuf.h
@@ -0,0 +1,11 @@
+--- src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h.orig	2020-10-27 08:02:11 UTC
++++ src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h
+@@ -44,6 +44,8 @@
+ #include <EGL/egl.h>
+ #include <EGL/eglext.h>
+ 
++#include <array>
++
+ // compatibility with libdrm <= 2.4.74
+ #ifndef DRM_FORMAT_RESERVED
+ #define DRM_FORMAT_RESERVED           ((1ULL << 56) - 1)
